scripts/faster_execution.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
```

Comment:
I really don't like the name `faster_execution.py`. Maybe `open_url_in_instance.py`?

In theory, we could also not have this script at all and just run the socket logic in the main qutebrowser binary as well (before importing anything from PyQt).

Reply:
I agree the name leaves something to be desired, but `open_url_in_instance` is worse because it doesn't even convey what the script is designed to do.  Qutebrowser *already* opens URLs in its own instance.  Why, then, do we need a script?  The name is confusing.

Do you have any other ideas?  How about `faster_open_in_instance`?

I mentioned incorporating this into qutebrowser proper in #4482 but consider doing so a major change (and therefore best made by @The-Compiler).  Please feel free to do so.

Reply:
I'm fine with the old name really - I think people got used to it, and I feel like it's okay (after all, it does open an URL in a qutebrowser instance - it doesn't necessarily need to describe why it's needed).

Reply:
As for making it part of qutebrowser itself: That doesn't make much sense to me - qutebrowser already has this functionality built in. It's slow because it does a lot of import (and other initialization) before that part of the code gets run, so if anything, that should be made faster (see #1098).